doc/en/CAS/STACK-Maxima_sandbox.md

```diff
@@ -4,17 +4,21 @@ It is very useful when authoring questions to be able to test out Maxima code in
 That is to say, to run a desktop version of Maxima with the local settings and STACK specific functions loaded.  This is also used in [reporting](../Authoring/Reporting.md) and analysis of students' responses.
 To do this you will need to load your local settings, and also the libraries of Maxima functions specific to STACK.
 
-# STACK - Maxima sandbox (without access to a server)
+## Setup Maxima and wxMaxima
+
+Install Maxima (http://maxima.sourceforge.net/) and wxMaxima (https://wxmaxima-developers.github.io/wxmaxima/).
+
```

Comment:
On my system (Linux Mint) in addition to installing Maxima I also had to install the maxima-share package.

Reply:
Gustav,
I think if we write parallel instructions on how to install maxima locally we will end up with outdated instructions which are wrong!  So, sorry this was a problem.

Reply:
@sangwinc I think it is important to point out that just installing from the link provided is not enough, but that STACK needs additional maxima packages. I agree that we should not go into details of how to install those extra packages, we just should draw the reader's attention to the fact that they are needed.